ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -296,6 +310,30 @@ public void tearDown() {
     REST_APP.getServiceContext().close();
   }
 
+  @Test
+  public void shouldStreamMultiplePushQueries() throws Exception {
```

Comment:
This works with and without my patch, but it doesn't hurt to have a test for it.